libs/config/src/http-common.config.ts

```diff
@@ -9,7 +9,8 @@ export interface IHttpCommonConfig {
 export default registerAs('http-common', (): IHttpCommonConfig => {
   const configs: JoiUtils.JoiConfig<IHttpCommonConfig> = JoiUtils.normalizeConfigNames({
     httpResponseTimeoutMS: {
-      label: 'HTTP_RESPONSE_TIMEOUT_MS',
+      value: process.env.HTTP_RESPONSE_TIMEOUT_MS &&
+        Math.max(Number(process.env.HTTP_RESPONSE_TIMEOUT_MS), Number(process.env.API_TIMEOUT_MS ?? 0)),
```

Comment:
This `Math.max` would break in the case where `API_TIMEOUT_MS` is set lower than `HTTP_RESPONSE_TIMEOUT_MS`, but `API_TIMEOUT_MS` should always be less than `HTTP_RESPONSE_TIMEOUT_MS`